packages/rnv/pluginTemplates/renative.plugins.json

```diff
@@ -2244,7 +2239,7 @@
     },
     "plugins": {
         "renative": {
-            "version": "0.28.21"
+            "version": "0.28.19-alpha.2"
```

Comment:
Downgraded?

Reply:
It needs to match the version of `/packages/renative/`

Reply:
Thinking about this a bit more - it's hard to know when version is best here as we're releasing 0.28 and 0.29 versions and develop can't track both? When `release/0.29` is merged back into develop it'll bump to correct version

Reply:
Looking at [this](https://github.com/pavjacko/renative/blob/master/packages/renative/package.json#L3) it can be reverted then I guess?

Reply:
actually, it didn't match the others so was a potential bug. Thanks